xtext/org.icyphy.linguafranca/src/org/icyphy/generator/GeneratorBase.xtend

```diff
@@ -866,10 +874,14 @@ abstract class GeneratorBase extends AbstractLinguaFrancaValidator {
      *  will never have a `-c` flag.
      */
     protected def compileCCommand(String fileToCompile, boolean doNotLinkIfNoMain) {
-        val cFilename = getTargetFileName(fileToCompile);            
-        val relativeSrcFilename = "src-gen" + File.separator + cFilename;
-        val relativeBinFilename = "bin" + File.separator + fileToCompile;
-
+    	val isBash = findCommandEnv(config.compiler)
```

Comment:
A better name for the variable `isBash` would be `env` or `environment`. Also, I noticed you indent with tabs. Please use 4 spaces per tab instead.

Reply:
I agree with these.